### PR TITLE
[PW-7302] Adding Retrieve3DS2/ClassicPayments and CardDetails/Checkout endpoints

### PR DESCRIPTION
**Pull request by jillingk**
**Description**
Added two endpoints that were missing on request of merchants.

**Tested scenarios**
UT

**Fixed issue**:

### CODE REVIEW

**Review by wboereboom — CHANGES_REQUESTED** [review]
Single comment on the tests. Looks good otherwise

File: Adyen/Model/Checkout/CardBrandDetails.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+//  ###### ######  #####( ######  #####. ######  #####          #####  ######
+//  #############  #############  #############  #############  #####  ######
+//   ############   ############  #############   ############  #####  ######
+//                                       ######
+//                                #############
+//                                ############
+// 
+//  Adyen Dotnet API Library
+// 
+//  Copyright (c) 2020 Adyen B.V.
```
Comment (AlexandrosMor) [inline Adyen/Model/Checkout/CardBrandDetails.cs:19]: Until we remove the headers lets add it to the correct year
```suggestion
//  Copyright (c) 2022 Adyen N.V.
```

File: Adyen/Model/Checkout/CardDetailsRequest.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+//  ###### ######  #####( ######  #####. ######  #####          #####  ######
+//  #############  #############  #############  #############  #####  ######
+//   ############   ############  #############   ############  #####  ######
+//                                       ######
+//                                #############
+//                                ############
+// 
+//  Adyen Dotnet API Library
+// 
+//  Copyright (c) 2020 Adyen B.V.
```
Comment (AlexandrosMor) [inline Adyen/Model/Checkout/CardDetailsRequest.cs:19]:
```suggestion
//  Copyright (c) 2022 Adyen N.V.
```

**Review by wboereboom — APPROVED** [review]

File: Adyen.Test/Adyen.Test.csproj [hunk 3]
```diff
@@ -46,6 +43,10 @@
     <ProjectReference Include="..\Adyen\Adyen.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Include="Mocks\checkout\card-details-success.json" />
```
Comment (AlexandrosMor) [inline Adyen.Test/Adyen.Test.csproj:47]: Why do we add this specific location in the project file?

File: Adyen/Service/Resource/Payment/Retrieve3DS2Result.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+//  * ###### ######  #####( ######  #####. ######  #####          #####  ######
+//  * #############  #############  #############  #############  #####  ######
+//  *  ############   ############  #############   ############  #####  ######
+//  *                                      ######
+//  *                               #############
+//  *                               ############
+//  *
+//  * Adyen Dotnet API Library
+//  *
+//  * Copyright (c) 2020 Adyen B.V.
```
Comment (AlexandrosMor) [inline Adyen/Service/Resource/Payment/Retrieve3DS2Result.cs:18]:
```suggestion
//  * Copyright (c) 2022 Adyen N.V.
```

**Review by AlexandrosMor — CHANGES_REQUESTED** [review]

**Review by AlexandrosMor — APPROVED** [review]

**Review by wboereboom — APPROVED** [review]

**Pull request merged**